internal/clients/github/client.go

```diff
@@ -656,3 +657,16 @@ func (c *Client) GetOrganizationSecrets(org string) (*gh.Secrets, error) {
 	}
 	return secrets, nil
 }
+
+func (c *Client) GetSecurityAndAnalysisForRepository(repo, owner string) (*gh.SecurityAndAnalysis, error) {
```

Comment:
check if we can add this to the graphql query

Reply:
Didn't find something in GraphQL that gives info about this